examples/camera-app/linux/src/clusters/camera-avstream-mgmt/camera-av-stream-manager.cpp

```diff
@@ -503,6 +509,10 @@ void CameraAVStreamManager::OnAttributeChanged(AttributeId attributeId)
             GetCameraAVStreamMgmtServer()->GetSoftLivestreamPrivacyModeEnabled());
         break;
     }
+    case HardPrivacyModeOn::Id: {
```

Comment:
This should not be here. This is for writable attributes that are changed.
The HardPrivacyMode can only be set out-of-band by the app-pipe in our case. But is otherwise a read-only attribute.

Reply:
We keep two copies of this field, one in the cluster server and one in the example app, I use out-of-band to write the HardPrivacyMode in the cluster server, then the changed valued should be updated to the app.

We don't have the API to write this attribute in the cluster server, since it is ready only.